### PR TITLE
Autofixture Rework of Fluent Test and Das Test

### PR DESCRIPTION
**Pull request by CodeWolf86**

### CODE REVIEW

File: src/SFA.DAS.ProviderRegistrations.UnitTests/Application/Queries/GetInvitationByIdQueryHandlerTests.cs [hunk 1]
```diff
@@ -1,69 +1,52 @@
...
-                    r2.Invitation.EmployerOrganisation == f.Invitation.EmployerOrganisation &&
-                    r2.Invitation.Ukprn == f.Invitation.Ukprn));
+            var result = await handler.Handle(query, new CancellationToken());
+            result.Should().BeNull();
         }
 
-        [Test]
-        public Task Handle_WhenHandlingGetInvitationByIdQueryAndInvitationIsNotFound_ThenShouldReturnNull()
+        [Test, ProviderAutoData]
+        public async Task Handle_WhenHandlingGetInvitationByIdQueryAndInvitationIsNotFound_ThenShouldReturnNull(
```
Comment (benagain) [inline src/SFA.DAS.ProviderRegistrations.UnitTests/Application/Queries/GetInvitationByIdQueryHandlerTests.cs]: `_ThenShouldReturnNull`?  

`result.Invitation.Should().NotBeNull();`? 😉

File: src/SFA.DAS.ProviderRegistrations.UnitTests/Application/Queries/GetInvitationByIdQueryHandlerTests.cs [hunk 2]
```diff
@@ -1,69 +1,52 @@
...
+            GetInvitationByIdQueryHandler handler,
+            GetInvitationByIdQuery query)
         {
-            return RunAsync(f => f.SetInvitation(), f => f.Handle(), (f, r) => r.Should().NotBeNull()
-                .And.Match<GetInvitationByIdQueryResult>(r2 =>
-                    r2.Invitation.EmployerOrganisation == f.Invitation.EmployerOrganisation &&
-                    r2.Invitation.Ukprn == f.Invitation.Ukprn));
+            var result = await handler.Handle(query, new CancellationToken());
+            result.Should().BeNull();
```
Comment (benagain) [inline src/SFA.DAS.ProviderRegistrations.UnitTests/Application/Queries/GetInvitationByIdQueryHandlerTests.cs]: Beautiful 💯

File: src/SFA.DAS.ProviderRegistrations.UnitTests/Application/Queries/GetInvitationByIdQueryHandlerTests.cs (hunk 1 above)
Comment (CodeWolf86) [reply src/SFA.DAS.ProviderRegistrations.UnitTests/Application/Queries/GetInvitationByIdQueryHandlerTests.cs]: That one was a late change, i copied the wrong previous test name to show how it is written differently. 😄

**Comment by CodeWolf86** [comment]
Closing PR, Rebasing off of branch further along with more unit tests and will recreate now we're pursuing AutoFixture

**Pull request closed without merging**